stories/Atom/Table/table.jsx

```diff
@@ -63,7 +63,9 @@ export const TableTag = ({ text, tdtext, details, variant, size, responsive, ...
       </table>
       )
       : (
-        <table className={[`${variant_options[`${variant}`]}`, `${variant_options1[`${size}`]}`, `${variant_options2[`${responsive}`]}`].join(` `)}>
+        <table className={[`${variant_options[`${variant}`]?`${variant_options[`${variant}`]} `:''}`,
```

Comment:
@harishkumar-srijan Define a `const` to filter the space in between multiple classes.
```
const cls = (...classes) => classes.filter(Boolean).join(' ');
```

And then change this code to
```
<table className={cls(`${variant_options[`${variant}`]}`, `${variant_options1[`${size}`]}`, `${variant_options2[`${responsive}`]}`)}>
```

You can use this normally as:
```
<div className={cls('foo', condition && 'bar')} />
// If condition satisfies then the above will render: 
// <div class="foo bar"></div>
```

Reply:
done